### ISSUE TITLE
French search page gives results in English search page

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#103]
URL: All - eg. https://hcdev.openplus.ca/en/
User Agent: Chrome, Firefox
Language: Both

Issue Description: When the user is in a French record and types in a search term on top right to search, the result is English search results page. It should keep the user in French.

Steps to reproduce: Go to a record (https://hcdev.openplus.ca/fr/avis-rappel/information-pour-consommateurs-au-sujet-nouilles-marque-maggi-provenant-inde), type in a search term on top right
https://hcdev.openplus.ca/fr/avis-rappel/information-pour-consommateurs-au-sujet-nouilles-marque-maggi-provenant-inde

The user is taken to English results
![image](https://user-images.githubusercontent.com/14280062/136616210-724d9df8-aa7b-4d44-8448-649e871702ad.png)

Expected results: It should stay in French

**Comment by echouin** [comment]
OP confirms FIXED

**Comment by ndolter** [comment]
Retested and the search results continue to be in English and for some reason there are a bunch of new issues on the page in step 1 that I don't think were there before.
Steps: 
1. Visit https://hcdev.openplus.ca/fr/avis-rappel/information-pour-consommateurs-au-sujet-nouilles-marque-maggi-provenant-inde.  Some new issues on this page:  missing translations, text overlap, 2 date modified, search filed not translated
![image](https://user-images.githubusercontent.com/68117281/137166033-a50ea263-e6ab-4bef-9c23-e33e4bcdb47c.png)

2. In the Search field in top right hand corner, search on 'rappel'
3. Page results continue to be in English when they should be in French

**Comment by davidgotlieb** [comment]
Hi @ndolter ,

I'm getting french results when I search., Although I do see the section headers on the filters on the right are in English so we'll look at that (unless those top two are the correct French and juts are the same word).

![image](https://user-images.githubusercontent.com/53194802/137167045-9c7a2031-8faf-4a05-bcf6-c1708a1102be.png)

**Comment by echouin** [comment]
@ndolter this is fixed for me as well?

**Comment by ndolter** [comment]
I just looked at it again (Firefox ver 93.0) and although some of the translations are now there, the issues with the overlapping text and two modification dates are still present.
![image](https://user-images.githubusercontent.com/68117281/137188675-5eff6d42-3c02-4e51-b53c-2551099c175c.png)

Validated that the issue about entering text in the Search field on the French page now correctly takes the user to a French page.

**Comment by davidgotlieb** [comment]
Thanks @ndolter The overlapping text is just because the image isn't there, which was explained in another ticket. That's just the reference for the database to pull the image once the system is updated. It won't show that way on the live server.

@echouin can you ask Craig about the double modification date? IT might just be wonky data in the existing system, in which case not much we can do, but could be a glitch. Maybe you can check and see if this is happening on any other old recalls?

**Comment by echouin** [comment]
@ndolter Issue noted to OP. They have asked that it be moved to our issue queue as it is not accessibility. Recommend this item be closed.

**Comment by ndolter** [comment]
Spot checked a few pages and the 2 modification dates and overlapping text is no longer an issue, however the Search page no longer has a Date modified.  While it isn't a WCAG issue, it is something that we as a Web standards team are mandated to check and it must pass as well. 
https://hcdev.openplus.ca/fr/recherche/site?search_api_fulltext=test&op=
![image](https://user-images.githubusercontent.com/68117281/138931632-42c5e57e-fb85-4569-bc67-5c90612e30cd.png)

**Comment by echouin** [comment]
@ndolter sent back to OP

**Comment by echouin** [comment]
@ndolter can you please confirm what is needed for WCAG? It appears as though additional items have been added?
If the primary WCAG issue is resolved, this item should be closed.

Thank you,

**Comment by ndolter** [comment]
Validated that the Date modified issue has now been resolved.  Search results are displayed correctly in French.
